pkg/workflow/manager.go

```diff
@@ -369,11 +370,22 @@ func (w *WorkflowManager) enqueueAfter(obj interface{}, d time.Duration) {
 	return
 }
 
+// enqueueKeyAfter enqueues a workflow key after a given time.
+// enqueueKeyAfter is non-blocking.
+func (w *WorkflowManager) enqueueKeyAfter(key string, d time.Duration) {
```

Comment:
We already have enqueueController function which does something very similar. Any reason we need both?